README.md

```diff
@@ -1,10 +1,170 @@
+
+## How to contribute
+
+### Adding a new virtualization component
+
+If you have an idea about a component that could be a great fit for rust-vmm,
+please open a
+[review request issue](https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=repository-review-request.md&title=Repository%2FCrate+Review+Request).
+The issue must provide the explanations and justifications behind the need for
+a new component, highlighting the purpose of this new repository, how will it
+be used by other VMMs or Hypervisors, and other information that you find
+relevant.
+
+Discussions about the proposal will happen on the same issue, and once an
+agreement is reached regarding the name (this part is hard!) and the design,
+an empty repository will be created in the rust-vmm GitHub organization.
+
+To add functionality you will need to send Pull Requests (PRs) against the
+newly created repository. While we do not expect the code to be perfect from
+the first try, we do want to keep a high standard in terms of quality for all
+existing repositories under rust-vmm.
+
+### Become a member of rust-vmm
+
+Submit a new
+[access request issue](https://github.com/rust-vmm/community/issues/new?assignees=andreeaflorescu%2C+jbryce%2C+sameo&labels=&template=access-request-to-rust-vmm-organization.md&title=Access+Request+to+rust-vmm).
+
+## rust-vmm development
+
+### Publishing on crates.io - Requirements list
+
+We consider crates published on crates.io to be production ready. In order to
+maintain a high security and quality standard, the following list of
+requirements must be checked before having the first version published:
+
+- **High level documentation**. This documentation should be added
+   to `src/lib.rs` and it will end up being the home page of the crate on
+   [docs.rs](https://docs.rs/). Here you should include information about what
+   is the crate suppose to do, how can it be used in other projects as well as
+   usage examples. Check out the
+   [kvm-ioctls docs](https://docs.rs/kvm-ioctls/0.1.0/kvm_ioctls/) for an
+   example.
+- **Documentation for the public interface**. Everything belonging to the
+  public interface of the crate must be properly documented. Where applicable,
+  the methods should also have usage examples. The code examples are
+  particularly useful because they are run when running the unit tests with
+  `cargo test`. It is a nice way to ensure that your comments and examples
+  don't go out of sync with the code. To make sure we don't miss anything this
+  rule is enforced by using `#![deny(missing-docs)]` in `src/lib.rs`.
+- **Unit Tests**.
+- **Integration Tests**. All components will eventually be tested using the
+  same common [Buildkite pipeline](https://buildkite.com/docs/pipelines). This
+  is still WIP. The tests include style checks and other lint checks, a
+  coverage test, builds with various configurations and on different
+  architectures and operating systems.
+- **README.md**
+- **LICENSE**
```

Comment:
Here we should specify our preference, i.e. what we agreed at the Denver PTG and in the mailing list: We prefer dual licensing our crates with Apache 2.0 OR BSD 3-Clause and describe why. And that this dual licensing is obviously a preference for crates code that is written from scratch. We are not asking for re-licensing existing code.